src/adapters/stargate/products/voting-escrow/metadata/ethereum.protocol-token.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "address": "0x0e42acBD23FAee03249DAFF896b78d7e79fBD58E",
+    "tag_id": "is_protocol_token",
+    "value": true
+  },
+  {
+    "address": "0x0e42acBD23FAee03249DAFF896b78d7e79fBD58E",
+    "tag_id": "token",
+    "value": {
+      "address": "0x0e42acBD23FAee03249DAFF896b78d7e79fBD58E",
+      "name": "veSTG",
+      "symbol": "veSTG",
+      "decimals": 18
+    }
+  },
+  {
+    "address": "0x0e42acBD23FAee03249DAFF896b78d7e79fBD58E",
+    "tag_id": "underlying_token_link",
+    "value": "0xAf5191B0De278C7286d6C7CC6ab6BB8A73bA2Cd6"
+  },
+  {
+    "address": "0xAf5191B0De278C7286d6C7CC6ab6BB8A73bA2Cd6",
+    "tag_id": "token",
+    "value": {
+      "address": "0xAf5191B0De278C7286d6C7CC6ab6BB8A73bA2Cd6",
+      "name": "Stargate Finance",
+      "symbol": "STG",
+      "decimals": 18
+    }
+  },
+  {
+    "address": "0xAF667811A7eDcD5B0066CD4cA0da51637DB76D09",
+    "tag_id": "underlying_token_link",
+    "value": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
+  },
+  {
+    "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+    "tag_id": "token",
+    "value": {
+      "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+      "name": "USD Coin",
+      "symbol": "USDC",
+      "decimals": 6
+    }
+  },
+  {
+    "address": "0xAF667811A7eDcD5B0066CD4cA0da51637DB76D09",
+    "tag_id": "is_reward_manager",
+    "value": true
+  }
+]
```

Comment:
Bit of a spike to try out the "tag" format.

Reply:
Let's start with this, we don't need to commit to anything specific yet as we can always regenerate it.